src/app/layouts/base-layout/components/footer/about-list.tsx

```diff
@@ -5,7 +5,7 @@ import { Logo } from '@/shared/ui/logo';
 const aboutList = [
   {
     title: 'About RS',
-    to: `/${ROUTES.COMMUNITY}/#about`,
+    to: `/${ROUTES.COMMUNITY}/#about-school`,
```

Comment:
We need somehow store this anchor links in constants

Reply:
Done